jekyll/_cci2/gh-bb-integration.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: classic-docs
+title: GiHub and Bitbucket Integration
+description: GiHub and Bitbucket Integration
+categories: [migration]
+Order: 60
+---
+
+This document provides an overview of the process of checking out your code from GitHub or Bitbucket and running your configuration. After you add a project to CircleCI CircleCI adds the following GitHub or Bitbucket settings to your project using the permissions you gave CircleCI when you signed up:
+- A **deploy key** that is used to check out your project from GitHub or Bitbucket.
+- A **service hook** that is used to notify CircleCI when you push to GitHub or Bitbucket.
+
+## Overview
+
+After you create and commit a `.circleci/config.yml` file to your GitHub or Bitbucket repository CircleCI immediately checks your code out and runs your first job along with any configured tests. For example, if you are working on a Rails project using Postgres specifications and features you might configure the following job run step:
+
+```
+jobs:
+  steps:
+    - run: |
+        bundle install
+        bundle exec rake db:schema:load
+        bundle exec rspec spec
+        bundle exec cucumber
+```
+        
+CircleCI runs your tests on a clean container every time so that your code is never accessible to other users and the test are fresh each time you push. Watch your tests update in real-time on [your dashboard](https://circleci.com/dashboard) or get status when CircleCI sends you a notification email after the job finishes.
+
+## Authorizing Multiple GitHub Repositories for Testing
```

Comment:
The paragraph below is correct. I was confused by the heading (it sounds like it's something you need to do if you have more than one repo you want to test on CircleCI). Perhaps change to "Enable Your Project to Check Out Additional Private Repositories"